src/client/pages/OfferNew/Checkout/index.tsx

```diff
@@ -158,18 +261,25 @@ export const Checkout = ({
   const [emailUpdateLoading, setEmailUpdateLoading] = useState(false)
   const [ssnUpdateLoading, setSsnUpdateLoading] = useState(false)
   const [isShowingFailModal, setIsShowingFailModal] = useState(false)
+  const [fakeLoading, setFakeLoading] = React.useState(false)
+  const [reallyLoading, setReallyLoading] = React.useState(false)
```

Comment:
Oh these are the ones @mehdielket removed in #671, no? I hope you're not gonna run into to a nasty merge conflict 🤭